### ISSUE TITLE
Fix request: false postive UnconditionalOperationOnLogArgument

### ISSUE THREAD

**Issue by stokpop** [jborgers/PMD-jPinpoint-rules#468]
False positive when there are more conditions in if statement:

```java
    public void testGood(String faultCode) {
        if (ids.contains("some-id") && log.isDebugEnabled()) {
            log.debug("Getting [{}]", faultCode.toUpperCase(Locale.US));  // good, but false positive
        }
    }
```

**Comment by stokpop** [comment]
There are more cases: `UnconditionalConcatInLogArgument` and `UnconditionalCreatedLogArguments`, also fix for these cases?

Note that for `&&` it is ok, but for `||` it is not always good. But that would make the rules even more complex?

**Comment by jborgers** [comment]
I think better for all rules that use the if isLogEnabled.

It should be &&, the || is a violation in my view.
